packages/server/src/async/producers/feed-items/member-joined-to-community.js

```diff
@@ -0,0 +1,73 @@
+const assert = require('assert');
+const _ = require('lodash');
+
+const { ProducerFeedItems } = require('./producer-feed-items');
+const { CHANNEL_MEMBERSHIP_TYPE } = require('../../../models/enums/channel-membership-type');
+
+const { Producer } = require('../producer');
+
+class MemberJoinedToCommunity extends ProducerFeedItems {
```

Comment:
Can this just be called ``MemberJoinedCommunity``? Or maybe it could go with the enum naming discussed above?

Reply:
Ok, lets without  "to", like